docs/content/en/docs/Coding Milestones/PoC2023q1/mailbox-controller.md

```diff
@@ -105,26 +111,37 @@ That should provoke logging like the following from the mailbox controller.
 I0305 18:07:20.490417   85556 main.go:369] "Created missing workspace" worker=0 mbwsName="niqdko2g2pwoadfb-mb-f99e773f-3db2-439e-8054-827c4ac55368"
 ```
 
-And you can verify that like so.
+And you can verify that as follows:
 
 ```console
-$ kubectl ws root:edge
-Current workspace is "root:edge".
+$ kubectl ws root:espw
+Current workspace is "root:espw".
 
-$ kubectl get Workspace
+$ kubectl get workspace
 NAME                                                       TYPE        REGION   PHASE   URL                                                     AGE
 niqdko2g2pwoadfb-mb-f99e773f-3db2-439e-8054-827c4ac55368   universal            Ready   https://192.168.58.123:6443/clusters/0ay27fcwuo2sv6ht   22s
 ```
 
-Next, `kubectl delete` that Workspace, and watch the mailbox
+Next, `kubectl delete` that workspace, and watch the mailbox
 controller wait for it to be gone and then re-create it.
 
 ```console
 I0305 18:08:15.428884   85556 main.go:369] "Created missing workspace" worker=2 mbwsName="niqdko2g2pwoadfb-mb-f99e773f-3db2-439e-8054-827c4ac55368"
 ```
 
-Finally, go back to your inventory workspace and `kubectl delete
-SyncTarget stest1` and watch the mailbox controller react as follows.
+Finally, go back to your inventory workspace:
```

Comment:
How about retaining the English description of what is about to be done?

Reply:
addressed with latest push, thx